src/explore-education-statistics-frontend/src/components/Link.tsx

```diff
@@ -40,24 +39,9 @@ const Link = ({
         className,
       )}
       data-testid={testId}
+      {...props}
```

Comment:
Similar to other comment - explicit props should go after the spread to avoid being accidentally overridden in the future i.e.

```tsx
<RouterLink
  {...props}
  href={to}
  prefetch={prefetch}
  // etc...
/>
```

Reply:
Good spot, fixed